docs/support_tickets/README.md

```diff
@@ -0,0 +1,48 @@
+# Placeholders
+
+* {customer_firstname}: The first name of the customer who submitted the review.
```

Comment:
@OverlordFX3 please put preview in same line as the placeholder, for example :

* {customer_firstname}: The first name of the customer who submitted the review. Preview example: Matt

its easier to scan this way document

Reply:
@emiliauskas-fuzzy updated